src/components/EditFrom/EditForm.jsx

```diff
@@ -89,6 +92,31 @@ export default function EditForm() {
 						}
 					/>
 				</StyledLabel>
+				<StyledLabel htmlFor="categories">Choose a category</StyledLabel>
+				<select
+					{...register('category', { required: 'This filed is required' })}
+					id="categories"
+				>
+					<option value="">{toBeEditedVideo?.category}</option>
+					{categories.map(category => (
+						<option key={category.id} value={category.name}>
+							{category.name}
+						</option>
+					))}
+				</select>
+				<ErrorMessage
+					errors={errors}
+					name="category"
+					render={({ messages }) =>
+						messages &&
+						Object.entries(messages).map(([type, message]) => (
+							<StyledInputWarning key={type} role="alert">
```

Comment:
Is the `type` you are using for the `key` always unique?

Reply:
Yes, should be. Its preset from React Hook Form

Reply:
Ahh good, thanks :)